kwave/options.py

```diff
@@ -2,11 +2,15 @@
 import os
 from dataclasses import dataclass, field
 from tempfile import gettempdir
-from typing import List, Optional
+from typing import List, Optional, TYPE_CHECKING
 
 import numpy as np
 
-from kwave.utils.io import get_h5_literals, get_date_string
+if TYPE_CHECKING:
```

Comment:
Hmm, I would prefer a solution without this :) Do we have any alternatives?

Reply:
this seems to be directly from the mypy docs:

https://mypy.readthedocs.io/en/stable/runtime_troubles.html#import-cycles

Reply:
Wow didn't expect this, alright seems good then :)